provision-mktg.sh

```
@@ -8,10 +8,11 @@ echo "CREATE DATABASE IF NOT EXISTS marketingsite;" | docker exec -i edx.devstac
 
 echo "** Marketing: Copy cacheed files to code dir **"
 docker compose `echo ${DOCKER_COMPOSE_FILES}` exec mktg bash -c 'cp -Rn /cache/* /app/.'
-docker compose `echo ${DOCKER_COMPOSE_FILES}` exec mktg bash -c 'pip3 install -r requirements.txt'
+echo "** Marketing: Install requirements **"
+docker compose `echo ${DOCKER_COMPOSE_FILES}` exec mktg bash -c 'pip install -r requirements.txt'
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)